### DISCUSSION TITLE
Feature Request: Fetch sponsors for multiple accounts

### DISCUSSION THREAD

**Discussion by mhanberg** [JamesIves/github-sponsors-readme-action#810] [Ideas]
It would be great if you could provide a list of PATs and it would fetch the sponsors from each of them and remove the duplicates, so that it shows as one big pool of sponsors

**Comment by JamesIves** [comment]
Thanks for the suggestion! What is the use case for such a feature?

You may be able to achieve something similar by simply running the action multiple times and parsing the data it outputs.

**Reply by mhanberg** [reply]
Hi!

The use case is we have an OSS project with an official team of 3 people and each has their individual sponsors. 

I'd like to be able to run the action once for all three of us and have it de duplicate sponsors that sponsor more than one of us. 

This might get funky with the functionality that lets you group by tier, as someone could sponsor each person for different amounts.
